frontend/src/routes/KitchenSink.tsx

```diff
@@ -3,6 +3,19 @@ import logo from '../images/mutiny-logo.svg';
 import init, { NodeManager, InitOutput } from "node-manager";
 
 function App() {
+    // https://stackoverflow.com/questions/47879864/how-can-i-check-if-a-browser-supports-webassembly
+    const wasmSupported = (() => {
+        try {
```

Comment:
this logic should happen inside a useEffect hook to make sure it doesn't get re-run on every re-paint (welcome to frontend!)
https://beta.reactjs.org/apis/react/useEffect

Reply:
I've noticed that node manager runs twice too. Does everything need to be in `useEffect` ?

Reply:
Moved it